src/reference/sectionC_interpretation.inc

```diff
@@ -7,13 +7,19 @@
 Interpretation of ``import`` elements
 -------------------------------------
 
-#. Each :code:`import` element present in a :ref:`CellML infoset<specA_cellml_infoset>` (the importing infoset) SHALL define a new and distinct instance of the CellML infoset which is referenced by the :code:`href` attribute (the imported infoset).
-   This imported infoset instance SHALL provide the context in which the child of an import element (a :code:`units` or :code:`component` element) is defined and interpreted.
+#. Each :code:`import units` or :code:`import component` element present in a :ref:`CellML infoset<specA_cellml_infoset>` (the importing infoset) SHALL define a new and distinct instance of the CellML infoset (the imported infoset) that is specified by the parent :code:`import` element's :code:`href` attribute.
 
-#. The imported infoset SHALL be considered such that items relevant to the imported infoset (:code:`unit` children of imported :code:`units` elements or the :ref:`encapsulated set<specC_encapsulation_set>` of imported :code:`component` elements) are themselves available to that :code:`import` element child.
-   It is noted, for the avoidance of doubt, that the importing infoset SHALL NOT have access to such implicitly imported items, but only to explicitly specified :code:`import` element children.
+#. A :ref:`units reference<specC_units_reference>` occurring within an imported component or within an imported units definition, SHALL be resolved with respect to the imported infoset.
 
-See :ref:`Units reference<specC_units_reference>` and :ref:`Component reference<specC_component_reference>` for the specifics of importing units and components.
+#. When determining the equivalent variable set of a variable in an imported component:
+
+   #. connections defined in the importing infoset shall be handled as described in :ref:`Interpretation of ``map_variables`` <specC_interpretation_of_map_variables>`.
```

Comment:
Does this formatting of the link build correctly?  I didn't think it would ... might have to change to:
```
:ref:`Interpretation of map_variables<specC_interpretation_of_map_variables>`
```
(no space before <, and no formatting allowed between the backticks)?

Reply:
Correct, the formatting of the link doesn't build correctly. What works is (done in commit e574934):
```
:ref:`Interpretation of map_variables<specC_interpretation_of_map_variables>`
```
(FWIW, whether there is a space of or not before `<` doesn't make any difference.)

Still, ideally, "map_variables" would be rendered using code style, i.e. "`map_variables`". I know this can be done with when we have an external link (see https://opencor.ws/developer/thirdPartyLibraries.html), but can't remember / don't know how to do it for a cross reference.